retriever/lib/get_opts.py

```diff
@@ -10,7 +10,7 @@
 
 module_list = MODULE_LIST()
 script_list = [module.SCRIPT.shortname for module in module_list]
-json_list = [module.SCRIPT.shortname for module in module_list
```

Comment:
hi @henrykironde `json_list` generated here is used as hints for `edit_json` and `delete_json`. Earlier it used to list all  SCRIPT.shortname. But the problem was, here https://github.com/weecology/retriever/blob/master/retriever/__main__.py#L101 we are checking user's input against all json's File name and not SCRIPT.shortname.
So argparse was hinting/forcing to use names like `bird-size` but it didn't match with any file, as file's name is `bird_size`. And when we try to use `bird_size`, argparse declares it as invalid. 
Here I have changed hints from Scripts.shortname to Script's Filename.